modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -35,30 +35,34 @@ spec:
   logLevel: Normal
   managementState: Managed
   operatorLogLevel: Normal
+  mode: Predictive                                     <1>
   profileCustomizations:
-    namespaces:                                        <1>
+    namespaces:                                        <2>
       excluded:
       - my-namespace
-    podLifetime: 48h                                   <2>
-    thresholdPriorityClassName: my-priority-class-name <3>
-  profiles:                                            <4>
+    podLifetime: 48h                                   <3>
+    thresholdPriorityClassName: my-priority-class-name <4>
+  profiles:                                            <5>
   - AffinityAndTaints
-  - TopologyAndDuplicates                              <5>
+  - TopologyAndDuplicates                              <6>
   - LifecycleAndUtilization
   - EvictPodsWithLocalStorage
   - EvictPodsWithPVC
 ----
 --
-<1> Optional: Set a list of user-created namespaces to include or exclude from descheduler operations. Use `excluded` to set a list of namespaces to exclude or use `included` to set a list of namespaces to include. Note that protected namespaces (`openshift-*`, `kube-system`, `hypershift`) are excluded by default.
+<1> Optional: By default, the descheduler does not evict pods. To evict pods, set `mode` to `Automatic`.
+<2> Optional: Set a list of user-created namespaces to include or exclude from descheduler operations. Use `excluded` to set a list of namespaces to exclude or use `included` to set a list of namespaces to include. Note that protected namespaces (`openshift-*`, `kube-system`, `hypershift`) are always excluded.
+<3> Optional: Enable a custom pod lifetime value for the `LifecycleAndUtilization` profile. Valid units are `s`, `m`, or `h`. The default pod lifetime is 24 hours.
+<4> Optional: Specify a priority threshold to consider pods for eviction only if their priority is lower than the specified level. Use the `thresholdPriority` field to set a numerical priority threshold (for example, `10000`) or use the `thresholdPriorityClassName` field to specify a certain priority class name (for example, `my-priority-class-name`). If you specify a priority class name, it must already exist or the descheduler will throw an error. Do not set both `thresholdPriority` and `thresholdPriorityClassName`.
+<5> Add one or more profiles to enable. Available profiles: `AffinityAndTaints`, `TopologyAndDuplicates`, `LifecycleAndUtilization`, `SoftTopologyAndDuplicates`, `EvictPodsWithLocalStorage`, and `EvictPodsWithPVC`.
+<6> Do not enable both `TopologyAndDuplicates` and `SoftTopologyAndDuplicates`. Enabling both results in a conflict.
+
 +
 [IMPORTANT]
```

Comment:
Your update here is moving this note to be at the bottom. It's actually supposed to go below the step that talks about namespace exclusion (callout 2 now). I had just added this note recently - can you change it back please?

Also something about this update is throwing off the formatting for the rest of the section:

![image](https://user-images.githubusercontent.com/7443887/220152474-c8f252bf-7581-4fbd-8e5b-66ed3e5386f3.png)

Reply:
@bergerhoffer Please review the latest updates.